### DISCUSSION TITLE
Difference between `0.weight` and `op.weight`

### DISCUSSION THREAD

**Discussion by ljleb** [hako-mikan/sd-webui-supermerger#117] [Q&A]
What is the difference between `0.weight` and `op.weight`, and between `0.bias` and `op.bias` in elemental merge? Do these layers have different behaviours, or are they the same type of layer in the SD architecture?

**Comment by miasik** [comment]
> What is the difference between `0.weight` and `op.weight`, and between `0.bias` and `op.bias` in elemental merge? Do these layers have different behaviours, or are they the same type of layer in the SD architecture?

I don't have any information about the elements, but I'm really interested in it. Where did you get your information about the elements difference?

**Comment by hako-mikan** [comment]
I don't fully understand it either, but I assume the element is a special one, corresponding to the exit of U-Net and having to do with the conversion of the data processed by U-Net to Latent.
